manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: iSpringSolutionsInc.FreeCam8
+PackageVersion: 8.7.27159
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral

```

```suggestion
  Architecture: x86
```
x86 will still install on both x86 and x64 devices.

---

hmm